torchbenchmark/models/doctr/__init__.py

```diff
@@ -0,0 +1,36 @@
+from doctr.models import ocr_predictor
+import numpy as np
+import torch
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+from typing import Tuple
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.DETECTION
+    DEFAULT_EVAL_BSIZE = 2
+
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+
+        self.model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True).to(self.device)
+        # fake document file
+        self.example_inputs = []
+        for _i in range(self.batch_size):
```

Comment:
Is it correct to set input batch size like this?

Reply:
looks good to me. @ryanchesler, Please, double-check when you have time